### PR TITLE
Added title attributes to iframe examples in 4.7.6

### PR DESCRIPTION
**Pull request by LJWatson**
Proposed fix for <a href="https://github.com/w3c/html/issues/298">#298</a>

### CODE REVIEW

**Comment by travisleithead** [comment]
LGTM!

File: sections/semantics-embedded-content.include [hunk 1]
```diff
@@ -3490,15 +3490,15 @@ My &lt;img src="heart.png" alt="heart"&gt; breaks.
   &lt;/footer&gt;
   &lt;article&gt;
   &lt;footer&gt; Thirteen minutes ago, &lt;a href="/users/ch"&gt;ch&lt;/a&gt; wrote: &lt;/footer&gt;
-  &lt;iframe seamless sandbox srcdoc="&lt;p&gt;did you get a cover picture yet?"&gt;&lt;/iframe&gt;
+  &lt;iframe title="Comment" seamless sandbox srcdoc="&lt;p&gt;did you get a cover picture yet?"&gt;&lt;/iframe&gt;
```
Comment (stevefaulkner) [inline sections/semantics-embedded-content.include:3493]: need to remove 'seamless' attribute from example

File: sections/semantics-embedded-content.include [hunk 2]
```diff
@@ -3490,15 +3490,15 @@ My &lt;img src="heart.png" alt="heart"&gt; breaks.
...
   &lt;/article&gt;
   &lt;article&gt;
   &lt;footer&gt; Nine minutes ago, &lt;a href="/users/cap"&gt;cap&lt;/a&gt; wrote: &lt;/footer&gt;
-  &lt;iframe seamless sandbox srcdoc="&lt;p&gt;Yeah, you can see it &lt;a href=&amp;quot;/gallery?mode=cover&amp;amp;amp;page=1&amp;quot;&gt;in my gallery&lt;/a&gt;."&gt;&lt;/iframe&gt;
+  &lt;iframe title="Comment" seamless sandbox srcdoc="&lt;p&gt;Yeah, you can see it &lt;a href=&amp;quot;/gallery?mode=cover&amp;amp;amp;page=1&amp;quot;&gt;in my gallery&lt;/a&gt;."&gt;&lt;/iframe&gt;
```
Comment (stevefaulkner) [inline sections/semantics-embedded-content.include:3497]: need to remove 'seamless' attribute from example

File: sections/semantics-embedded-content.include [hunk 3]
```diff
@@ -3490,15 +3490,15 @@ My &lt;img src="heart.png" alt="heart"&gt; breaks.
...
   &lt;/article&gt;
   &lt;article&gt;
   &lt;footer&gt; Five minutes ago, &lt;a href="/users/ch"&gt;ch&lt;/a&gt; wrote: &lt;/footer&gt;
-  &lt;iframe seamless sandbox srcdoc="&lt;p&gt;hey that's earl's table.
+  &lt;iframe title="Comment" seamless sandbox srcdoc="&lt;p&gt;hey that's earl's table.
```
Comment (stevefaulkner) [inline sections/semantics-embedded-content.include:3501]: need to remove 'seamless' attribute from example

File: sections/semantics-embedded-content.include [hunk 4]
```diff
@@ -3872,7 +3872,7 @@ My &lt;img src="heart.png" alt="heart"&gt; breaks.
   &lt;h1&gt;Mirror Mirror&lt;/h1&gt;
   &lt;p&gt;Part of the MovieInfo™ Database&lt;/p&gt;
   &lt;nav&gt;
-  &lt;iframe seamless src="nav.inc"&gt;&lt;/iframe&gt;
+  &lt;iframe title="Movie links" seamless src="nav.inc"&gt;&lt;/iframe&gt;
```
Comment (stevefaulkner) [inline sections/semantics-embedded-content.include:3875]: need to remove 'seamless' attribute from example

**Comment by travisleithead** [comment]
Looks like the merge cleaned this up. However, found one more occurrence. filed https://github.com/w3c/html/issues/328 to take care of it.

**Pull request merged**